indoor-floor-level-tracker/README.md

```diff
@@ -64,7 +64,7 @@ setting up your hardware.
 
 The following hardware is required to run the Indoor Floor-Level Tracker.
 
-* [Feather Starter Kit for Swan](https://shop.blues.io/products/notecarrier-swan)
+* [Blues Starter Kit](https://shop.blues.io/products/blues-starter-kit-for-north-america)
```

Comment:
Oh, hey, this is a great time for me to bring up a feature request! I want a single shop link I can use for a “Blues Starter Kit”, because I don’t love assuming North America and we’re going to need to link to this—a lot.

Take it or leave it, but you gave me the perfect opportunity to ask 🙂